pkg/services/postgresql/dbms_types.go

```diff
@@ -9,15 +9,14 @@ type dbmsProvisioningParameters struct {
 
 func (
 	d *dbmsManager,
-) getProvisionParametersSchema() map[string]*service.ParameterSchema {
-
+) getProvisionParametersSchema() map[string]service.ParameterSchema {
 	p := getDBMSCommonProvisionParamSchema()
-
-	p["alias"] = &service.ParameterSchema{
-		Type:        "string",
-		Description: "Alias to use when provisioning databases on this DBMS",
-		Required:    true,
-	}
+	aliasSchema := service.NewSimpleParameterSchema(
```

Comment:
Should have caught this in the first iteration of this feature...

I see the above repeated in a few places. Shouldn't "alias" be dealt with once wherever location and resource group are dealt with? (Those are conditionally required if a service doesn't have a parent. Alias should be conditionally required if a service doesn't have any children.)

Reply:
Yeah, this is a good idea! Should have handled that in the first pass through here. Moving that up to the service is a great idea.